boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.yaml

```diff
@@ -21,3 +21,5 @@ supported:
   - dma
   - gpio
   - i2c
+  - spi
+  - hs_spi1
```

Comment:
where is this tag used?

Reply:
Deleted it since it's not used currently.